xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/WebHome.xml

```diff
@@ -94,18 +129,20 @@
         <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
       </description>
     </class>
-    <name>Menu.WebHome</name>
-    <number>0</number>
-    <className>AppWithinMinutes.LiveTableClass</className>
-    <guid>0168db3f-33fe-45e4-9a6d-ce31e6e5f66b</guid>
     <property>
       <class>Menu.MenuClass</class>
     </property>
     <property>
       <columns>doc.name doc.date doc.author _actions</columns>
     </property>
     <property>
-      <description>This is a simple application that helps you create navigation menus to be displayed either horizontally as a top bar after the page header or vertically in a side panel.</description>
+      <dataSpace/>
+    </property>
+    <property>
+      <description>{{velocity}}
+$services.localization.render('menu.description')
+{{/velocity}}
```

Comment:
You might want to consider using the `translation` macro since there is no other work justifying the need for the `velocity` macro.

Example:
```
{{translation key="some.translation"/}}
```
...as mentioned on http://extensions.xwiki.org/xwiki/bin/view/Extension/Localization/

Reply:
Yes translation macro should always be preferred when possible.